grim.go

```diff
@@ -27,7 +27,9 @@ func (i *Instance) SetConfigRoot(path string) {
 }
 
 // PrepareGrimQueue creates or reuses the Amazon SQS queue named in the config.
-func (i *Instance) PrepareGrimQueue() error {
+// If the grimServerID is over 15 chars, it is truncated and logged
+// The message logged will be the truncMessage param, or the default if truncMessage is blank
+func (i *Instance) PrepareGrimQueue(truncMessage string, logger *log.Logger) error {
```

Comment:
Why are you injecting truncMessage as parameter?  Just define it as a constant somewhere and get rid of this complexity.

Also you don't need to have the comment about the truncation in the godoc header.